plugins/hydra_submitit_launcher/example/config.yaml

```diff
@@ -0,0 +1,6 @@
+# @package _group_
```

Comment:
primary config files does not need a package statement (they default to `_global_`).